python/pyspark/streaming/dstream.py

```diff
@@ -23,6 +23,8 @@
 
 if sys.version < "3":
     from itertools import imap as map, ifilter as filter
+else:
+    long = int
```

Comment:
Can you add a test for it? Seems only used once and shouldn't be difficult to add a test.

Reply:
Adding a call to slice in `tests.py` should be enough to test this. I'm surprised we haven't caught this before, but I suppose this isn't a very frequently exercised code path.